### PR TITLE
add the url for user to complete registration

### PR DESCRIPTION
**Pull request by martineehrlich**
## Description
We want to return the url that redirects to the user registration page with the entityId generated by the Relayer.

## Todos
- [x] Tests
- [x] Documentation
- [x] Link to Trello

screen shot 2019-02-06 at 4 25 04 pm

### CODE REVIEW

**Comment by martineehrlich** [comment]
[Show registration url when using sparkswap register](https://trello.com/c/znHgfqyN/1124-show-registration-url-when-using-sparkswap-register)

File: broker-cli/proto/broker.proto [hunk 1]
```diff
@@ -152,6 +152,8 @@ message GetIdentityResponse {
 message RegisterResponse {
   // the Identifier assigned by the Relayer for the Entity associated with the provided Public Key
   string entity_id = 1;
+  // url for registering a user
```
Comment (treygriffith) [inline broker-cli/proto/broker.proto]: URL to complete user registration

File: broker-cli/commands/register.spec.js [hunk 2]
```diff
@@ -58,6 +58,6 @@ describe('register', () => {
   it('logs the output from the broker', async () => {
     await register(args, opts, logger)
     expect(infoSpy).to.have.been.calledOnce()
-    expect(infoSpy).to.have.been.calledWith('Successfully registered public key with relayer')
+    expect(infoSpy).to.have.been.calledWith(`Successfully registered public key with Sparkswap Relayer. Go to ${url.cyan} to complete registration.`)
```
Comment (treygriffith) [inline broker-cli/commands/register.spec.js]: Any way to make this a bit more lively or branded?

File: broker-cli/proto/broker.proto [hunk 3]
```diff
@@ -263,7 +265,8 @@ service AdminService {
   }
 
   /**
-   * Register takes the publicKey of users identity returns the entityId of the entity created by the Relayer
+   * Register takes the publicKey of users identity returns the entityId of the entity
+   * created by the Relayer and the url for the user to complete registration
```
Comment (treygriffith) [inline broker-cli/proto/broker.proto]: Need to update the output of the shell and javascript in this file

File: broker-daemon/broker-rpc/admin-service/register.js [hunk 4]
```diff
@@ -15,8 +23,8 @@ async function register ({ relayer, logger }, { RegisterResponse }) {
   const { entityId } = await relayer.adminService.register({ publicKey })
 
   logger.info('Successfully registered public key with relayer', {entityId})
-
-  return new RegisterResponse({ entityId })
+  const url = `${REGISTER_URL}/${entityId}`
```
Comment (treygriffith) [inline broker-daemon/broker-rpc/admin-service/register.js]: [nit] I think the slash should probably be part of the constant, or you should do a `join` with the slash.

**Pull request merged**